src/utils/apiKey.ts

```
@@ -0,0 +1,102 @@
+import jwt from 'jsonwebtoken';
+import crypto from 'crypto';
+import { findHashedKeyInDb } from '../database/models/ApiKey';
+
+const JWT_SECRET = process.env.JWT_SECRET!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)